src/App.js

```diff
@@ -1,23 +1,19 @@
-import logo from './logo.svg';
+import React from 'react'
 import './App.css';
-
+import BlogPage from './Component/BlogPage'
+import Navbar from './Component/Navbar'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
```

Comment:
Use "organize imports" in your code editor. First all third party deps should be imported, then the local dependencies and then Css files should be imported last